### DISCUSSION TITLE
INF [RTSP] [session 601416432] destroyed (no UDP packets received in a while)

### DISCUSSION THREAD

**Discussion by LittleRain626** [bluenviron/mediamtx#1123] [Questions]
本地推流失败
ffmpeg -re -stream_loop -1 -i Infrared.mp4 -c copy -f rtsp rtsp://localhost:8554/mystream

INF [RTSP] [session 601416432] destroyed (no UDP packets received in a while)

**Comment by aler9** [comment]
Switch to TCP by using the `-rtsp_transport` flag:

```
ffmpeg -re -stream_loop -1 -i Infrared.mp4 -c copy -f rtsp -rtsp_transport tcp rtsp://localhost:8554/mystream
```

[a comment by an automated account is not shown]
